components/netty-http-echo-service/src/main/java/org/wso2/performance/common/netty/echo/EchoHttpServer.java

```diff
@@ -57,6 +67,9 @@
     @Parameter(names = "--enable-ssl", description = "Enable SSL")
     private boolean enableSSL = false;
 
+    @Parameter(names = "--keyStore-file", description = "Keystore file")
```

Comment:
Rename this to `--key-store-file`. Also validate the file. Eg: https://github.com/chrishantha/performance-apim/blob/master/components/jwt-generator/src/main/java/org/wso2/performance/apim/microgw/jwt/JWTGenerator.java#L73-L75